.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+.hintrc
+.stylelintrc.json
```

Comment:
- [ ] kindly remove the `.hintrc` and `.stylelintrc.json` from the gitignore file, as these files are meant to be in your file root directory as part of your linters environment setup